docs/administration/upgrade/to_v6.0.adoc

```diff
@@ -297,17 +332,18 @@ spec:
       authentication:
         token:
           from: secret
-        secret:
-          key: token
-          secretName: logcollector-token
+          secret:
+            key: token
+            name: logcollector-token
     tls:
       ca:
         key: service-ca.crt
-        configMapName: openshift-service-ca.crt
+        secretName: logcollector-token
```

Comment:
Replace with the change here https://github.com/openshift/cluster-logging-operator/pull/2761